src/cache-restore.ts

```diff
@@ -21,6 +21,7 @@ export const restoreCache = async (
     throw new Error(`Caching for '${packageManager}' is not supported`);
   }
   const platform = process.env.RUNNER_OS;
+  const arch = process.env.RUNNER_ARCH;
```

Comment:
I think we can use `os.arch()` or use architecture from [here](https://github.com/actions/setup-node/blob/2a017f350dbf6c4b6bb4508cc83809719115162e/src/main.ts#L22-L35).

Reply:
How about `RUNNER_OS` in line 23? Maybe we should keep the code consistent

Reply:
Hi @pengx17, Could you please change the line to use os.arch() or use the arch input. We have to ensure that code works for all type of runners here.

Also please sync your code to use main branch and rebuild the code to pass all the checks.

Reply:
done